tests/AbstractLoopTest.php

```diff
@@ -398,6 +398,35 @@ function () {
         $this->loop->run();
     }
 
+    public function testSignal()
+    {
+        if (!extension_loaded('pcntl')) {
+            $this->markTestSkipped('Signal test skipped because PCNTL extension isn\'t loaded.');
```

Comment:
It could… though if an app depends on signals, likely it will throw anyway, so may as well throw from the `onSignal()` method. Note that UV supports signal handling without the need of the `pcntl` extension.